rllib/models/tf/attention_net.py

```diff
@@ -0,0 +1,287 @@
+"""
```

Comment:
Can you add a brief comment about this net and its state (e.g., experimental, a bit slow due to duplicate history stored in h)?

Reply:
+1